lms/djangoapps/certificates/api.py

```diff
@@ -228,8 +229,13 @@ def certificate_downloadable_status(student, course_key):
         'is_unverified': True if current_status['status'] == CertificateStatuses.unverified else False,
         'download_url': None,
         'uuid': None,
+        'is_invalid_cert': False,
```

Comment:
Actually, on second thought, We can have `is_invalid` data field here. But at the same time, we should have the function `cert_generation_enabled` to return false when the student have their certificate invalidated.